### PR TITLE
hafner/#1739-send-device-info

### PR DESCRIPTION
**Pull request by coleHafner**
- Added device sensor (this emits SIM info and some other stuff that Henrik implemented)
- Alphabetized `package.json` commands
- Merged in `saga`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,11 +53,21 @@ An accelerometer recording can be found [here](./__tests__/sensors/accelerometer
...
 A thermometer recording can be found here [here](./__tests__/sensors/thermometer-recording.txt).
 
+### Device samples
+
+A device info sample can be found here [here](./__tests__/sensors/device-info-recording.txt).
+
 ## Creating Devices and Certs
 
-Clone this repo and install the dependencies:
+Install the dependencies and run simulator.
```
Comment (coderbyheart) [inline README.md]: This section was about **Creating Devices and Certs**.

Please add the information on how to run this simulator locally to the [Usage](https://github.com/nRFCloud/device-simulator/blob/aedb03025d67ce79bdb9c9c77a98ffbddbbfb2f5/README.md#usage) section.

File: package.json [hunk 2]
```diff
@@ -6,17 +6,18 @@
   "typings": "dist/index.d.ts",
   "bin": "dist/cli.js",
   "scripts": {
-    "precompile": "npm run schemas2ts",
-    "build": "npm run lint && npm run compile",
-    "compile": "npm run compile:ts",
-    "compile:ts": "tsc",
-    "lint": "npm run tslint && npm run prettier:lint",
-    "tslint": "tslint --project ./tsconfig.json",
+	"build": "npm run lint && npm run compile",
```
Comment (coderbyheart) [inline package.json]: Indentation is off, please correct according to rest of file.

File: schemas/app.schema.json [hunk 3]
```diff
@@ -13,7 +13,8 @@
             "enum": [
                 "GPS",
                 "FLIP",
-                "TEMP"
+				"TEMP",
```
Comment (coderbyheart) [inline schemas/app.schema.json]: Indentation is off, please correct according to rest of file.

File: src/app/AppModel.ts [hunk 4]
```diff
@@ -9,7 +9,7 @@
  * Message format for simulator app
  */
 export interface AppMessage {
-  appId: 'GPS' | 'FLIP' | 'TEMP';
+  appId: 'GPS' | 'FLIP' | 'TEMP' | 'DEVICE';
```
Comment (coderbyheart) [inline src/app/AppModel.ts]:
```suggestion
  appId: 'GPS' | 'FLIP' | 'TEMP' | 'DEVICE_INFO';
```

`DEVICE` is too generic.

**Comment by coleHafner** [comment]
Device is too vague, you're right. I could change this now, but it would be meaningless unless we get the FW side to change first. It does us no good to have a simulator that doesn't simulate the actual FW in use. 

Let me work with Henrik and the rest of the FW team and then I will revisit this PR.

File: README.md (hunk 1 above)
Comment (coleHafner) [reply README.md]: Okay. Will do.

File: src/app/AppModel.ts (hunk 4 above)
Comment (coleHafner) [reply src/app/AppModel.ts]: Agreed. See comment.

**Comment by coleHafner** [comment]
> documentation of the app message in https://github.com/nRFCloud/application-protocols

Did not know that. Will update. Thanks.

File: README.md (hunk 1 above)
Comment (coleHafner) [reply README.md]: Right. That got jumbled when merging `saga`, overlooked the un-jumble before creating this PR. Fixed in f6f6e20

File: package.json (hunk 2 above)
Comment (coleHafner) [reply package.json]: Fixed in f6f6e20

File: schemas/app.schema.json (hunk 3 above)
Comment (coleHafner) [reply schemas/app.schema.json]: Fixed in f6f6e20

**Pull request merged**